peopleops/index.md

```diff
@@ -24,6 +24,19 @@ Sick leave, or having limited availibility is not recorded currently. Keep your
 manager updated on your health, and let them know what FlowForge can do for you
 to aid in your recovery.
 
+## Parental Leave
+
+If you live in a country where a statutory Parental Leave benefit is available,
+you will be required to claim statutory Parental Leave pay (if you are eligible).
+
+Whilst statutory parental leave will vary from country-to-country, at FlowForge
+we have a mandatory 4-week, full-pay, Parental Leave policy. Any employee is
```

Comment:
My only hesitation, what if people want a distraction from a baby? Maybe mandatory is to firm, and we should go with:

```suggestion
we advise you to take 4-weeks, full-pay, Parental Leave policy. Any employee is
```

Reply:
Given we already have an unlimited leave policy how is this 4 weeks any different?

Reply:
Maybe flip it around...

> Any employee can take up to 6-weeks is they wish without requiring manager approval. We recommend a minimum of 4 weeks, but recognise its a personal choice to make. Further leave can be requested, but should be discussed with your manager

Reply:
@sammachin As per handbook, anything over 2 weeks needs approval. This is an exception to that - parental leave means you have up to 6 weeks without needing approval.

Reply:
@knolleary I think your proposal is a good improvement and we should update the text to that.